ext/openssl/openssl.c

```diff
@@ -7939,11 +7939,10 @@ PHP_OPENSSL_API zend_string* php_openssl_random_pseudo_bytes(zend_long buffer_le
 	PHP_OPENSSL_CHECK_LONG_TO_INT_NULL_RETURN(buffer_length, length);
 	PHP_OPENSSL_RAND_ADD_TIME();
 	if (RAND_bytes((unsigned char*)ZSTR_VAL(buffer), (int)buffer_length) <= 0) {
+		php_openssl_store_errors();
```

Comment:
question (not for this PR): should we call it for few potentially failed BIO_free calls ? e.g. openssl_csr_export_to_file.

Reply:
Can BIO_free actually fail? If so, how?

Reply:
to be honest, I do not think so looking at the code paths and our cases. However checks are done here and there, e.g. openssl_x509_export_to_file, just a "consistency" thing whether we should remove or not.

Reply:
I'll leave that for someone more familiar with the ins & outs of the OpenSSL API, like Jakub.